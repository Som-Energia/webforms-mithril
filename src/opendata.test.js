var m = require('mithril')
var tidy = require('mithril-jest').tidy
function MyComponent() {
    return {
        view: function (vnode) {
            return m("div", vnode.attrs.text)
        }
    }
}
describe("Snapshot Example", function() {
    test("Positive control", function() {
        var c = m(MyComponent,{text: "Bye!"})
        const html = tidy(c);
        expect(html).toMatchSnapshot()
    })
    test("Negative control", function() {
        var c = m(MyComponent,{text: "Hello!"})
        const html = tidy(c);
        expect(html).toMatchSnapshot()
    })
})