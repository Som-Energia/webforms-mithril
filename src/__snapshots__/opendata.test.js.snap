// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Example Negative control 1`] = `
"<div>
  Nooo!
</div>
"
`;

exports[`Snapshot Example Positive control 1`] = `
"<div>
  Bye!
</div>
"
`;
